docs/spec/channel.md

```diff
@@ -391,6 +384,23 @@ Channels SHOULD expose a variety of metrics, including, but not limited to:
 Metrics SHOULD be enabled by default, with a configuration parameter included to
 disable them if desired.
 
+The Channel MUST recognize and pass through all tracing information from sender
+to subscribers using [W3C Tracecontext](https://w3c.github.io/trace-context/),
+although internally it MAY use another mechanism(s) to propagate the tracing
+information. The Channel SHOULD sample and write traces to the location
+specified in
+[`config-tracing`](https://github.com/knative/eventing/blob/master/config/config-tracing.yaml).
+
+Spans emitted by the Broker SHOULD follow the
+[OpenTelemetry Semantic Conventions for Messaging Systems](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/messaging.md)
+whenever possible. In particular, spans emitted by the Broker SHOULD set the
+following attributes:
+
+- messaging.system: "knative"
+- messaging.destination: url to which the event is being routed
+- messaging.protocol: the name of the underlying transport protocol
+- messaging.message_id: the event ID
+
```

Comment:
something we wanna check in conformance tests ?

Reply:
Cool, will update the test using the broker conformance as a reference

Reply:
done. I didn't add a check for `message.protocol` as that wasn't being tested in the broker either.